lib/amazon-pricing/definitions/instance-type.rb

```diff
@@ -116,6 +116,14 @@ def self.populate_lookups
       end
     end
 
+    def self.disk_bytes_per_sec_capacity(api_name)
+      PER_SEC_CAPACITIES[api_name][0] * 1024 * 1024
```

Comment:
is PER_SEC_CAPACITIES stores in megabits or megabytes?  that impacts whether its `1024` or `1000` right?